utils/utils.py

```
@@ -34,6 +34,24 @@ def compute_time_for_update(update_hour: str) -> Tuple[datetime, float]:
     return target_time, (target_time - now).total_seconds()
 
 
+def compute_time_for_notifications(update_hour: str) -> Tuple[datetime, float]:
+    """Return the seconds for the next notifications"""
+    try:
```

Same as previous try block, there are no added value as raise the exception again

---

still dont understand, the try block tries to read a file, if fails log it and continue. Did I do smt wrong ?

---

In Python, `raise e` raises the exception, it stops the function and propagates the exception `e`. 
In this case, the `try` block effectively does nothing more than if it were not present.

If you want the program to continue instead of stopping, you can log the error instead:

```python
except ValueError as e:
    logger.warning(f"{e}")
```

Or you can ignore the error entirely:

```python
except ValueError as e:
    pass
```

---

see my other comment.

---

Late, but done